### PR TITLE
Completed:

### PR DESCRIPTION
**Pull request by YamilNasser**
Python:
Mini Capstone
Lab 9

HTML:
Lab 1
Lab 2
Lab 3
Lab 4
Lab 5

### CODE REVIEW

**Comment by brucestull** [comment]
[Lab 12: Mini Capstone](https://github.com/PdxCodeGuild/class_swordfish/blob/main/1%20Python/labs/12%20Mini-Capstone.md)
**Completed**

File: code/yamil/html/lab_4/index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+                Tortilla: <input type="radio" name="tortilla" value="white_flour" checked>White Flour
+                <input type="radio" name="tortilla" value="wheat_flour">Wheat Flour
+                <input type="radio" name="tortilla" value="spinach">Spinach
+                <input type="radio" name="tortilla" value="corn">Corn
+                <br><br>
+                Meat:<input type="radio" name="meat" value="carnitas" checked>Carnitas
+                <input type="radio" name="meat" value="chicken">Chicken
+                <input type="radio" name="meat" value="sofritas">Sofritas
+                <br><br>
+                Extras:<input type="checkbox" name="extras" value="guacamole" checked>Guac
```
Comment (brucestull) [inline code/yamil/html/lab_4/index.html:32]: * The 'name's need to be different for each checkbox.
* [HTML \<input type="checkbox"> - www.w3schools.com](https://www.w3schools.com/tags/att_input_type_checkbox.asp)

File: code/yamil/html/lab_4/index.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    <div class="columns">
+        <div class="column">
+            <img src='burritos.jpg' style=width:15em;height:20em/>
+        </div>
+        <div class="column">
+            <h3>What would you like in your burrito?</h3>
+            <form action="https://webhook.site/0a7b5be8-8584-40c1-9ec7-005acfd2a691" method="post">
+                <input type="text" name="user_name"/> Name<br><br>
+                <input type="password"/> Password<br><br>
+                Tortilla: <input type="radio" name="tortilla" value="white_flour" checked>White Flour
```
Comment (brucestull) [inline code/yamil/html/lab_4/index.html:23]: * Reference:
    * [HTML \<input type="radio"> - www.w3schools.com](https://www.w3schools.com/tags/att_input_type_radio.asp)

File: code/yamil/html/lab_5/app.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
+from flask import Flask, render_template, request
```
Comment (brucestull) [inline code/yamil/html/lab_5/app.py:1]: * Application gives incorrect result:
    * See attached image:
image

**Comment by YamilNasser** [comment]
Revised and Completed:
Python Lab 9
HTML Lab 4
HTML Lab 5

**Review by brucestull — CHANGES_REQUESTED** [review]
Python [Lab 9: ATM](https://github.com/PdxCodeGuild/class_swordfish/blob/main/1%20Python/labs/09%20ATM.md)
**Completed**

[Lab 4: Burrito Order Form](https://github.com/PdxCodeGuild/class_swordfish/blob/main/2%20Flask%20%2B%20HTML%20%2B%20CSS/labs/04%20Burrito%20Order%20Form.md)
**Needs Work**
* Comments:
    * Delivery Instructions are currently not being passed to the API.
        * The form element needs an attribute in order to pass the text area's value to the API.
    * Use a `label` element to label the Delivery Instructions element.
        * The text included between the `textarea`s opening and closing tags, in the template, are incorrect. It forces the user to either delete that text or type over it. Use a `placeholder` attribute for text to be displayed for the user. Use a `label` element instead.

[Lab 5: Flask Redo](https://github.com/PdxCodeGuild/class_swordfish/blob/main/2%20Flask%20%2B%20HTML%20%2B%20CSS/labs/05%20Flask%20Redo.md) 
**Completed**

**Review by brucestull — APPROVED** [review]
[Lab 4: Burrito Order Form](https://github.com/PdxCodeGuild/class_swordfish/blob/main/2%20Flask%20%2B%20HTML%20%2B%20CSS/labs/04%20Burrito%20Order%20Form.md)
**Completed**

**Pull request merged**